### DISCUSSION TITLE
Welcome to dotnet-etcd Discussions!

### DISCUSSION THREAD

**Discussion by shubhamranjan** [shubhamranjan/dotnet-etcd#73] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MatthewComtois** [comment]
Hi, my name is Matthew and I’m new to ETCD. I’m trying to implement a leader election algorithm and I have a few question.

I do a put with a Lease, is the lease TTL in seconds or milliseconds? 

For the renewal of the lease, if my callback method is called, does that mean that the renewal worked?

Also, I’m creating a watcher to observe a delete, is their any way to create a watcher only for delete?
And lastly, if i create a watcher, do I need to create a new one after it has called my callback or the watcher will continue to listen to other event?

thank you
